.github/workflows/tests.yml

```
@@ -34,7 +34,7 @@ jobs:
       matrix:
         os: [ubuntu-20.04]
         floatx: [float64]
-        python-version: ["3.10"]
+        python-version: ["3,7", "3.10"]
```

You have a comma instead of dot. I think that may be the problem

```suggestion
        python-version: ["3.7", "3.10"]
```